ic-helper.ps1

```diff
@@ -99,6 +101,10 @@ function Find-GitRepositories {
             }
         }
     }
+
+    if (!$foundRepository) {
+        Write-Host -ForegroundColor Yellow "No repository was found in this directory: $path"
```

Comment:
Edit the message to `No Git repositories were found in directory $path`. I would also use white font color in this case.